docs/checks/docs_link_checker.py

```diff
@@ -338,15 +346,24 @@ def check_file(self, file: str) -> List[LinkReport]:
 )
 @click.option("--skip-external", is_flag=True)
 def scan_docs_click(
-    path: str, docs_root: Optional[str], site_prefix: str, skip_external: bool
+    path: str,
+    docs_root: Optional[str],
+    static_root: str,
```

Comment:
Hi @Rachel-Reverie! Since `static_root` is a required param, you'll need to add a `@click.option(<option name etc goes here>)` decorator to the method for the `static_root` param. Something like:
```
@click.option(
    "--static-root",
    "-sr",
    default= <what should the default be?>,
    help="Some helpful message describing what this option does"
)
```
I don't recall if it needs to be in the same order as the method params for click (I think it might be) but I would put it between the `--docs-root` and `--site-prefix` options anyway just to be consistent.

You'll also need to pass this param in the invoke method. I'll point that out in another comment.